### PR TITLE
feat(stats-detectors): New metric for duration

### PR DESCRIPTION
**Pull request by udameli**
We want to materialize a new less granular metric that will only include transaction name and duration.

The goal is to run timeseries analysis and find regressions in duration for specific transaction names. This will be used in Statistical Detectors so we'll need to fetch timeseries faster than we currently do. We're not interested in any other tags.

Cardinality analysis results were captured [here](https://getsentry.atlassian.net/browse/REQ3S-54)

Relay PR: https://github.com/getsentry/relay/pull/2513

### CODE REVIEW

File: src/sentry/sentry_metrics/indexer/strings.py [hunk 1]
```diff
@@ -87,6 +87,8 @@
     "d:transactions/alert@none": PREFIX + 133,
     "s:transactions/alert@none": PREFIX + 134,
     "g:transactions/alert@none": PREFIX + 135,
+    "t:transactions/duration@millisecond": PREFIX + 136,
+    "t:transactions/measurements.lcp@millisecond": PREFIX + 137,
```
Comment (udameli) [inline src/sentry/sentry_metrics/indexer/strings.py:91]: what's the significance of the prefix here? I've made it `t` just to make it unique but I'm not sure if it should be something else

**Review by obostjancic — APPROVED** [review]

File: src/sentry/sentry_metrics/indexer/strings.py [hunk 2]
```diff
@@ -87,6 +87,7 @@
     "d:transactions/alert@none": PREFIX + 133,
     "s:transactions/alert@none": PREFIX + 134,
     "g:transactions/alert@none": PREFIX + 135,
+    "d:transactions/duration_coarse@millisecond": PREFIX + 136,
```
Comment (wmak) [inline src/sentry/sentry_metrics/indexer/strings.py:90]: Ahh sorry I should've said this in our meeting, but I wonder if we should name this more explicitly? In starfish we have `exclusive_time_light` but I'm thinking if we add another "light" metric the name instantly becomes bad.

**Review by wmak — APPROVED** [review]
lgtm, minus a naming question

File: src/sentry/sentry_metrics/indexer/strings.py (hunk 2 above)
Comment (udameli) [reply src/sentry/sentry_metrics/indexer/strings.py:90]: hmmm `duration_low_cardinality`?

Comment (wmak) [reply src/sentry/sentry_metrics/indexer/strings.py:90]: lol i'm a big fan of very long descriptive names so I'm tempted to say like
`duration_with_only_transaction_tag` but maybe you can get more feedback from the relay team in https://github.com/getsentry/relay/pull/2513

Comment (udameli) [reply src/sentry/sentry_metrics/indexer/strings.py:90]: I'll check with @jan-auer when he's back

**Review by evanh — APPROVED** [review]
Pending Jan's feedback this is fine from the SnS side.

File: src/sentry/sentry_metrics/indexer/strings.py (hunk 2 above)
Comment (udameli) [reply src/sentry/sentry_metrics/indexer/strings.py]: ok yeah we've landed on `duration_light` to match `exclusive_time_light` based on Jan's feedback

**Pull request merged**